docs/embeddable-uis/element/access-request.mdx

```diff
@@ -0,0 +1,421 @@
+---
+sidebar_position: 3
+title: Access Request
+---
+
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Access Request Element
+
+<span className="subtitle-custom">
+The Access Request feature allows users to request access to restricted resources. The request is sent to the workspace owner, who can approve or deny it.
+</span>
+<br/>
+<br/>
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/ar-element.png" alt="ar-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+Access Request gives us new capability in our app. Now the users can ask to dynamically change their own permissions if they want to access a forbidden action.
+
+**For example**
+
+A normal user can't write a blog article, but an editor can. When a normal user wants to write an article, he requests access to the screen. When the system administrator gets his access request (in the user management element), he decides whether to grant him access or not.
```

Comment:
What is a "normal" user ? 
This term is not clear for a Permit.io user. 

Also, we should have a better use-case - for example, there is a user that want to create a post in a blog......
The example should be more as story than a technical

Reply:
Also, under the example, we should add more information about the element itself. 
1. How does it solve the use case above.
2. This feature allows you to embed the Access Request element to protect resources in your application
3. The user without the permissions will be able to request access using the element, that you will embed in your application where you want to protect your resources.
4. Later on, the administrator of the specific resource will be able to approve or deny the access request using the User Management Element.

Reply:
Also it would be really nice to have a diagram of the access request flow here (something easy with excalidraw or draw.io).